trade-finance-manager-ui/server/api-response-types/amendments.ts

```diff
@@ -0,0 +1,10 @@
+type ErrorResponse = {
+  status: number;
+  data: string;
+};
+
+type FacilityEndDateResponse = {
+  facilityEndDate: string;
+};
+
+export type getLatestCompletedAmendmentFacilityEndDate = ErrorResponse | FacilityEndDateResponse;
```

Comment:
Nitpick but type aliases should be pascal case

Reply:
Thanks have fixed!